src/skin/github-api.ts

```diff
@@ -80,4 +83,38 @@ export class GitHubApi implements Query {
     }
     return res;
   }
+
+  async markAsRead(user: GitHubUser, notificationIdToMarkAsRead: string): Promise<boolean> {
```

Comment:
```suggestion
  async markAsRead(user: GitHubUser, notificationIdToMarkAsRead: string): Promise<boolean> {
```

`notificationIdToMarkAsRead: string`は余りに長すぎだと思う
notificationIdとかでも十分伝わると思う

Reply:
22efdfe で対応